examples/spark2/src/main/scala/org/apache/carbondata/examples/PreAggregateDataMapExample.scala

```diff
@@ -175,7 +175,7 @@ object PreAggregateDataMapExample {
 
     // create pre-aggregate table by datamap
     spark.sql("""
-       CREATE datamap preagg_avg on table personTable using 'preaggregate' as
+       CREATE datamap preagg_avg_dm on table personTable using 'preaggregate' as
```

Comment:
why this modification is needed?

Reply:
As datamap name is a global, there is already the same  datamap name before.